lib/msf/core/payload/windows/reverse_winhttp.rb

```diff
@@ -117,9 +123,22 @@ def asm_reverse_winhttp(opts={})
     retry_count       = [opts[:retry_count].to_i, 1].max
     verify_ssl        = nil
     encoded_cert_hash = nil
-    encoded_url       = asm_generate_wchar_array(opts[:url])
+    encoded_uri       = asm_generate_wchar_array(opts[:uri])
     encoded_host      = asm_generate_wchar_array(opts[:host])
 
```

Comment:
Been pondering this a bit. There's a lot of config-based duplication all over this stuff, including the existing stagers. I'm thinking that I should instead create another PR after this that tidies up a bunch of stuff, and not just for these two stagers.

Are you cool with that @bcook-r7 ?